### ISSUE TITLE
)edit without FRICASEDITOR set opens more in Xterm but closes immediately 

### ISSUE THREAD

**Issue by gvanuxem** [fricas/fricas#154]
When using ')edit' without FRICASEDITOR set 'more' in an Xterm is launched but closes immediately:

(1) -> )edit license

Warning: environment variable FRICASEDITOR not set.
Launching 'more' in an 'xterm'.
Press space to continue, 'q' to quit.
(1) ->

- Greg

PS: I have working xterm and more.

**Comment by hemmecke** [comment]
I only see this issue, if I type ")edit FILE" where FILE is not existing. So maybe you have to add the full path to the file you want to see.

**Comment by gvanuxem** [comment]
> I only see this issue, if I type ")edit FILE" where FILE is not existing. So maybe you have to add the full path to the file you want to see.

Yes, you're right, with the full path that works perfectly. In fact, I have not noticed, but license is a directory. But same thing happened with the FriCAS INSTALL file when I am in the FriCAS source tree. So, the full path seems mandatory. Maybe an improvement can be done here, like the ')compile' system command. I will try to change the aim of the issue.
Thanks Ralf.

**Comment by hemmecke** [comment]
I am not completely sure, but it might be a problem with the value of DEFAULT(X)EDITOR in SPADEDIT.
```
DEFAULTXEDITOR='xterm -e "more +$line $name"'
```
In my installation a full path is not necessary. The only issue is if the file doesn't exist.
Maybe SPADEDIT should test the existence of the file before it starts the editor.

**Comment by oldk1331** [comment]
Fixed by 02a015e458b0d32c97287b904ffbd4cad661f845 .
